alpha/scripts/utils/setEntryProperties.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+if($argc < 3)
+{
+    echo "Arguments missing.\n\n";
+    echo "Usage: php " . __FILE__ . " {mapping} {isDateString} <realrun / dryrun> \n";
+    exit;
+}
+$mapping = $argv[1];
+$isDateString = ($argv[2] === "true");
+$dryRun = ($argv[3] === "dryrun");
+
+require_once(__DIR__ . '/../bootstrap.php');
+
+
+KalturaStatement::setDryRun($dryRun);
+
+$entryMappings = file($mapping, FILE_IGNORE_NEW_LINES);
+
+$properties = [""];
+$counter = 0;
+entry::setAllowOverrideReadOnlyFields(true);
+foreach ($entryMappings as $entryMapping)
+{
+    if($counter++ == 0){
```

Comment:
@TalDubovKaltura  I'd suggest just taking this if statement out of the loop altogether